### PR TITLE
fix: remove GA from cookie policy

### PR DESCRIPTION
**Pull request by iamacook**
## Overview
As part of our [analytics improvement](https://github.com/gnosis/safe-react/issues/3408), we no longer use GA and we therefore need not list details of their cookies.

### CODE REVIEW

File: src/pages/cookie.tsx
Comment (iamacook) [inline src/pages/cookie.tsx:443]: Fortmatic wallet is disabled on all chains. Their tracking won't occur.

**Comment by iamacook** [comment]
Please hold off on merging this.

File: src/pages/cookie.tsx
Comment (iamacook) [reply src/pages/cookie.tsx:443]: I've just double-checked: `detectedWallet` doesn't detect Fortmatic either.

**Review by DiogoSoaress — CHANGES_REQUESTED** [review]
Isn't the `_gac_<property-id>` cookie also part of GA?

**Review by DiogoSoaress — APPROVED** [review]
🧹  nice!

**Comment by francovenica** [comment]
Nothing to QA here. Just lets be sure this is not changed in prod until we have the new analytics implementation.

**Review by germartinez — CHANGES_REQUESTED** [review]
Be careful with this. The landing page itself is also using Google Analytics. That should be replaced before merging this PR.

**Comment by iamacook** [comment]
> Be careful with this. The landing page itself is also using Google Analytics. That should be replaced before merging this PR.

Maybe this an unnecessary change for the time being then. @johannesmoormann, do we intend to implement GTM on the landing page?

**Comment by johannesmoormann** [comment]
For now, the scope of the change is only for the web app, so let's leave GA in cookie policy.

**Pull request closed without merging**
